### DISCUSSION TITLE
previewFeature is Deprecated ? how can i solve this issue ?

### DISCUSSION THREAD

**Discussion by watashi407** [prisma/prisma#17976] [Q&A]
previewFeature is Deprecated ? how can i solve this issue ?

**Comment by mcbethio** [comment]
Hey @watashi407,

I'm afraid we're going to need more information in order to help you. Please can you send us some reproduction steps (what are you doing), code snippets, schema file and prisma version you are using?

Thanks

**Comment by watashi407** [comment]
3.3x above doesnt support referentialAction
but how can i solve this another problem ?
[image: image.png]

**Reply by nurul3101** [reply]
@watashi407 It looks like you tried to attach an image, but the image isn't included. Can you maybe retry adding the error image?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
